src/h5web/metadata-viewer/AttributeValueLink.tsx

```diff
@@ -0,0 +1,46 @@
+import { Fragment } from 'react';
+import { FiChevronsRight } from 'react-icons/fi';
+import styles from './MetadataViewer.module.css';
+
+interface Props {
+  onFollowPath: (value: string) => void;
+  value: unknown;
+}
+
+function AttributeValueLink(props: Props) {
+  const { onFollowPath, value } = props;
+
+  if (Array.isArray(value)) {
+    return (
+      <div className={styles.attrValueWrapper}>
+        [
+        {value.map((val, i) => (
+          // eslint-disable-next-line react/no-array-index-key
+          <Fragment key={i}>
+            <AttributeValueLink onFollowPath={onFollowPath} value={val} />
+            {i < value.length - 1 && <span>,</span>}
+          </Fragment>
+        ))}
+        ]
+      </div>
+    );
+  }
+
+  if (typeof value === 'string' && value !== '.') {
+    return (
+      <button
+        className={styles.attrValueBtn}
+        type="button"
+        aria-label={`Inspect ${value}`}
+        onClick={() => onFollowPath(value)}
+      >
+        {value}
+        <FiChevronsRight />
+      </button>
+    );
+  }
+
+  return <>{JSON.stringify(value)}</>;
```

Comment:
No need for the fragment in theory.

Reply:
I cannot remove it as I get `string | Element cannot be used as Element` when using `AttributeValueLink` recursively above.